### ISSUE TITLE
Script is outdated

### ISSUE THREAD

**Issue by evmjunkie** [spalladino/flashbots-unstake-and-transfer#1]
Error: bad response (status=400, headers={"date":"Fri, 03 Dec 2021 12:55:12 GMT","content-type":"application/json","content-length":"145","connection":"close","x-amzn-requestid":"b5b62059-8d18-44cf-a0c8-84568c8b40a6","x-amz-apigw-id":"JxgDjF1uiYcF6Uw=","x-amzn-trace-id":"Root=1-61aa13b0-0f85b0112c166092570b5e86;Sampled=0","cf-cache-status":"DYNAMIC","expect-ct":"max-age=604800, report-uri=\"https://report-uri.cloudflare.com/cdn-cgi/beacon/expect-ct\"","report-to":"{\"endpoints\":[{\"url\":\"https:\\/\\/a.nel.cloudflare.com\\/report\\/v3?s=bcNOErrf4NOyLZdM4dIdIOWcMqx3AfIkdzty0t69Oo5Affp0mp%2B3joe103JjI6xiCEmqADjVYVoyAmchMqIiTgap4LFgkRgAYeVusc3QSYsU7N6snmEVogtYXimXQWRZ1pjtY3U%3D\"}],\"group\":\"cf-nel\",\"max_age\":604800}","nel":"{\"success_fraction\":0,\"report_to\":\"cf-nel\",\"max_age\":604800}","server":"cloudflare","cf-ray":"6b7cf2aa8c2568fd-FRA"}, body="{\"error\":\"v0.1 bundle received, expected v0.2. See https://docs.flashbots.net/flashbots-auction/searchers/advanced/rpc-endpoint for latest docs\"}"

flashbots rpc needs to get JSON v2.0 RPC request, this script sends V1.0 requests.

**Comment by spalladino** [comment]
You're right, I've just added a notice on the README. Thanks for the heads up!

**Comment by evmjunkie** [comment]
You mind working together on a new script? It would be great for all those guys who get scammed on suspicious token listing-platforms and losing their money on scammers.

**Comment by spalladino** [comment]
Agree, but to be honest I don't have the time availability do work on this now. I can review and merge a PR though, as long as it includes some testing (even if manual).

**Comment by Vicopem01** [comment]
Is there no work around till now?

Implementing the v0.2 would be a lifesaver

**Comment by metacopy** [comment]
Do script needs many changes to get work? Very needed...
